cypress/integration/fetch_items.js

```diff
@@ -42,6 +42,21 @@ describe("Show list of items", function() {
       cy.get('.purchasedItem');
       expect(cy.contains('Purchased Item'));
     });
+
+    it('Verifies that all four items categories are present on the list view', function() {
+      cy.visit('/list');
+      expect(cy.contains('Soon Items' && 'Kind of Soon Items' && 'Not Soon Items' && 'Inactive Items'));
+    });
+
+    it('Are items being categorized correctly', function() {
+      cy.visit('/add');
+      cy.get('.inputField').type('salad2');
+      cy.get('#soonButton').click({ force: true });
```

Comment:
What does this `force` do?

Reply:
Without `force`, the Cypress throws an error. It says that the button element is not visible, and suggests using `force`. The test fails without it and passes with it, though I'm not certain why the element wasn't visible in the first place.